src/lib/sidenav/sidenav.ts

```diff
@@ -275,48 +313,74 @@ export class MdSidenavLayout implements AfterContentInit {
     }
   }
 
-  /** TODO: internal */
+  /** @override */
```

Comment:
Is there something else we can put here to indicate that it's being implemented as part of an interface? The rest of the TODO: internal's we prefixed with an underscore, but we can't do that here.